public/app/features/dashboard-scene/panel-edit/PanelEditor.tsx

```diff
@@ -154,6 +146,7 @@ export function buildPanelEditScene(dashboard: DashboardScene, panel: VizPanel):
         direction: 'column',
         primary: new SceneFlexLayout({
           direction: 'column',
+          minHeight: 200,
```

Comment:
PanelEdit css is a bit broken, when you open it with many queries there is no room for visualization (and you can't move the splitter, or scroll queries)

Reply:
Yes, @kaydelaney is already aware and will look into this :)